### ISSUE TITLE
Flags enums should be sorted

### ISSUE THREAD

**Issue by verdie-g** [DotNetAnalyzers/StyleCopAnalyzers#3796]
I would like to suggest a new rule that, with an enum like that
```csharp
[Flags]
public enum AttrCapabilities
{
    AllowView = 1 << 0,
    AllowCreate = 1 << 1,
    AllowChange = 1 << 2,
    AllowFilter = 1 << 4,
    AllowSort = 1 << 8,
}
```
would transform that code
```csharp
class Person
{
    [Attr(Capabilities = AttrCapabilities.AllowChange | AttrCapabilities.AllowFilter | AttrCapabilities.AllowView)]
    public string? FirstName { get; set; }
}
```
to
```csharp
class Person
{
    [Attr(Capabilities =  AttrCapabilities.AllowView | AttrCapabilities.AllowChange | AttrCapabilities.AllowFilter)]
    public string? FirstName { get; set; }
}
```

My use-case is that all properties of my domain objects are marked with these flags and it would help visually scanning them if they were sorted.

**Comment by sharwell** [comment]
I believe this rule would be better implemented in a separate analyzer project.
